sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/utils/HttpUtil.java

```diff
@@ -75,6 +75,24 @@ public static String getUserAgentPrefix() {
                        .orElse(DEFAULT_USER_AGENT_VALUE_PREFIX);
     }
 
+    public static String post(String url, Map<String, String> headers, String body, String contentType) {
```

Comment:
How about call `post(String url, Map<String, String> headers, String body, String contentType)` in `post(String url, String body, String contentType)`?